aip/0126.md

```diff
@@ -0,0 +1,102 @@
+---
+aip:
+  id: 126
+  state: reviewing
+  created: 2019-07-24
+permalink: /126
+redirect_from:
+  - /0126
+---
+
+# Enumerations
+
+It is common for a field to only accept or provide a discrete and limited set
+of values. In these cases, it can be useful to use enumerations (generally
+abbreviated "enums") in order to clearly communicate what the set of allowed
+values are.
+
+## Guidance
```

Comment:
Possibly add something around being explicit about which enums may change over time, as that impacts client coders?

Reply:
Done.